code/modules/mob/living/simple_animal/hostile/pitbull.dm

```diff
@@ -0,0 +1,50 @@
+/mob/living/simple_animal/hostile/pitbull
+	name = "pitbull"
+	icon_state = "pitbull"
+	icon_living = "pitbull"
+	icon_dead = "pitbull_dead"
+	speak_chance = 5
+	emote_hear = list("growls", "barks")
+	response_help = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm = "hits"
+
+
+	health = 25
+	maxHealth = 25
+	turns_per_move = 5//I don't know what this does
+	speed = 4
+	move_to_delay = 3
+
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	attacktext = "bites"
+	attack_sound = 'sound/weapons/bite.ogg'
+	faction = "wizard"
+
+	//var/mob/creator
+
+/mob/living/simple_animal/hostile/pitbull/New()
+	..()
+	desc = pick(
+		"There is no such thing as a bad dog, only bad owners.",
+		"Blame the owner not the breed!",
+		"My dog would never do that.",
+		"He's a big baby at heart.",
+		"Man's best friend.",
+		"Over 4 million pitbulls did not kill or hurt anyone today. Stop the myth.",
+		"Good with kids")
+
+/mob/living/simple_animal/hostile/pitbull/death()
+	..(TRUE)
+	var/mob/my_wiz = pitbulls[src]
+	pitbulls_count_by_wizards[my_wiz]--
+	pitbulls[src] = null
+	pitbulls -= src
+
+/mob/living/simple_animal/hostile/pitbull/ListTargets()
```

Comment:
you should be able to set the pitbull's `faction` var to the `\ref[]` of the wizard instead of doing any of this

Reply:
I was trying to exclude the master from being a target but I didn’t quite do it right.
maybe what I can do is have a life check to specifically turn on the master kind of like how goats get aggressive

Reply:
I just told you how to exclude the master from being a target

Reply:
yeah I saw and I plan to implement it. I was trying to answer kurf's question while also adding the idea of a life check that makes the pitbull turn specifically on the owner.